src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
+    public static final Random RANDOM = new Random();
+    public static final Color CIRCLE_COLOR = Color.WHITE;
+    public static final int CIRCLE_SIZE = 10;
+    public static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
+
+    public Figure getRandomFigure(int maxSize) {
+        String color = COLOR_SUPPLIER.getRandomColor();
+        int randFigureIndex = RANDOM.nextInt(AvailableFigure.values().length);
+        AvailableFigure figure = AvailableFigure.values()[randFigureIndex];
+        switch (figure) {
+            case RECTANGLE:
+                int randRightSide = RANDOM.nextInt(maxSize);
+                int randTopSide = RANDOM.nextInt(maxSize);
+                return new Rectangle(color, randTopSide, randRightSide);
```

Comment:
```suggestion
                return new Rectangle(color, randomSide(), andomSide());
```

Create separate method randomSide() for RANDOM.nextInt(maxSize);